open_academy/views/session_views.xml

```diff
@@ -10,4 +10,42 @@
 
     <menuitem id="session_menu" name="Sessions" sequence="1"
     action="session_action" parent="open_academy_root_session_menu"/>
+
+    <record id="session_view_form" model="ir.ui.view">
+        <field name="name">session.view.form</field>
+        <field name="model">session</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet string="Session">
+                    <group col="4">
+                        <field name="name" />
+                        <field name="start_date" />
+                        <field name="duration" />
+                        <field name="seats" />
+                        <field name="percent_seats" />
+                        <field name="active" />
+                        <field name="instructor_id" />
+                        <field name="course_id" />
+                        <newline/>
+                        <field name="attendee_ids" colspan="4" />
```

Comment:
Place this field in a separate group or within a notebook/page.
Also remove colspan, and if you choose the second option, place `nolabel="True"`.